### DISCUSSION TITLE
eb surface thermal condition

### DISCUSSION THREAD

**Discussion by jagmohas** [AMReX-Combustion/PeleC#610] [Q&A] [answered]
I see that by default PeleC sets these thermal conditions for the eb surfaces (line 58, 59 in pelec_defaults.H)

amrex::Real PeleC::eb_boundary_T = 1.0;
bool PeleC::eb_isothermal = true;

But according to this discussion, by default eb surfaces are adiabatic.
https://github.com/AMReX-Combustion/PeleC/discussions/313

Where do we overwrite the default eb surface thermal conditions to be adiabatic? In short, how do I make eb surfaces to be adiabatic in my simulation?

**Comment by hsitaram** [comment, marked as answer]
I was of the impression that it was adiabatic, but you are right, the
default is true! There may be a chance the code changed since.

If you want to do adiabatic eb wall, add this to your input/run script

pelec.eb_isothermal = 0
